recipes/wxwidgets/all/conanfile.py

```diff
@@ -393,6 +399,7 @@ def library_pattern(library):
                                        'comctl32',
                                        'ole32',
                                        'oleaut32',
+                                       'imm32',
```

Comment:
Seems to be conditional somehow, but I don't know where `WXMSW` is coming from or what it means

https://github.com/wxWidgets/wxWidgets/blob/db3e54b3dc5a4af812dea6e3945a2339ab01cbe2/build/cmake/lib/stc/CMakeLists.txt#L198

Reply:
It is coming from [wxWidgets/.../toolkit.cmake](https://github.com/wxWidgets/wxWidgets/blob/master/build/cmake/toolkit.cmake#L48). So the lib is required when we are building the windows toolkit.

Reply:
If I'm not missing something we don't have an option in this Conan recipe for enabling/disabling this toolkit. So it seems to be fine this way